.travis.yml

```diff
@@ -25,10 +27,10 @@ install:
   - conda config --add channels mosdef
   - conda create -n test-environment python=$PYTHON_VERSION --file requirements.txt
   - source activate test-environment
-  - pip install -e .
+  - python setup.py install
 
 script:
-  - py.test -v --cov=mbuild  --cov-report= --pyargs mbuild
+  - python -m pytest -v --cov=mbuild  --cov-report= --pyargs mbuild
```

Comment:
Ditto here

Reply:
I thought so, however, our OSX builds are still failing. It is related to `pytest`. These were ways to attempt to pin down the cause for failure.

A SIGABRT is happening here and although the tests pass on my machine, we cannot figure out what is causing this to fail on Travis.

https://travis-ci.org/mosdef-hub/mbuild/jobs/293851164
Here is a link to one of these instances. Have you encountered this before?

Reply:
Brief googling seems to hint at issues specific to python3.5 on travis OSX, perhaps memory related.